src/event.rs

```diff
@@ -2,16 +2,16 @@ use serde::de::DeserializeOwned;
 use serde::Serialize;
 
 #[cfg(not(feature = "upcasting"))]
-pub trait Event: Serialize + DeserializeOwned {}
+pub trait Event: Serialize + DeserializeOwned + Send + Sync {}
```

Comment:
Remove SendSync from here, these boundsshould be imposed where used (in postgres)